src/components/ChoiceGroup/ChoiceGroup.tsx

```diff
@@ -0,0 +1,112 @@
+/*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
+import '../../yamui';
+import * as React from 'react';
+import { BaseComponentProps } from '../../util/BaseComponent/props';
+import { ChoiceGroup as FabricChoiceGroup, IChoiceGroupOption } from 'office-ui-fabric-react/lib/ChoiceGroup';
+import { join } from '../../util/classNames';
+import Text from '../Text';
+import './ChoiceGroup.css';
+
+export interface ChoiceGroupOption {
+  /**
+   * A required key to uniquely identify the option.
+   */
+  key: string;
+
+  /**
+   * The label for the option.
+   */
+  label: string;
+
+  /**
+   * The sublabel for the option
+   */
+  sublabel?: string;
```

Comment:
if we keep with our other similar implementations this should probably be named "text"
https://github.com/Microsoft/YamUI/blob/master/src/components/Dropdown/Dropdown.tsx#L17

actually label shoud be optional and text should be required. would be consistent with Dropdown, and looks like label isn't always included in the mockups

Reply:
Ok - i will bring this inline with Dropdown